pom.xml

```diff
@@ -150,6 +152,28 @@
             <version>${spring-boot.version}</version>
         </dependency>
 
+        <!-- Kafka -->
```

Comment:
remove comments

Reply:
But I use this comments like tags for searching...

Reply:
you have a group id and artifact id of the dependency